DOCS/man/options.rst

```diff
@@ -1905,9 +1905,12 @@ Audio
 
     .. admonition:: Warning
 
-        There is not much reason to use this. HDMI supports uncompressed
-        multichannel PCM, and mpv supports lossless DTS-HD decoding via
-        FFmpeg's new DCA decoder (based on libdcadec).
+        In general it is not recommended to use passthrough, since HDMI
+        supports uncompressed multichannel PCM, and mpv supports lossless
+        DTS-HD decoding via FFmpeg's new DCA decoder (based on libdcadec).
+        
+        Nevertheless there are some cases in which it must be activated for
+        correct playback, such as 3D audio formats like Dolby Atmos or DTS:X.
```

Comment:
```suggestion
        Generally, passthrough is not recommended because HDMI supports
        uncompressed multichannel PCM, and mpv supports lossless DTS-HD decoding.

        However, there are cases where passthrough must be enabled for proper
        playback, particularly for 3D audio formats such as Dolby Atmos or DTS:X.
```
How about something like this? Particularity mentioning "new" (now almost 10 years old) decoder is not important detail.